<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReturnBooks</class>
 <widget class="QDialog" name="ReturnBooks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>592</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Return books</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* ======== MAIN WINDOW ======== */
QWidget {
    background-color: #f2eecb;
    font-size: 14px;
    color: #42522b;
}

/* ======== LABELS ======== */
QLabel {
    font-size: 14px;
    font-weight: bold;
    color: #42522b;
}

#titleLabel {
font-size: 36px;
}

/* ======== LINE EDITS ======== */
QLineEdit {
    background: #ffffff;
    border: 2px solid #b5c58b;
    border-radius: 10px;
    padding: 6px;
    color: #42522b;
}

QLineEdit:focus {
    border: 2px solid #7d8f52;
    background: #ffffff;
}

/* ======== BUTTONS ======== */
QPushButton {
    background-color: #7d8f52; 
    color: white;
    border-radius: 12px;
    padding: 8px 18px;
    font-size: 15px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #6e8048;
}

QPushButton:pressed {
    background-color: #5c6c3b;
}

/* SECONDARY BUTTON (Clear) */
#clearButton {
    background-color: #d6cfa8;
    color: #3c3c3c;
}

#clearButton:hover {
    background-color: #c7bf9b;
}

#clearButton:pressed {
    background-color: #b3aa87;
}

/* BACK BUTTON */
#goBackButton_5 {
    background: transparent;
    border: none;
    color: #42522b;
    font-size: 16px;
}
QPushButton#goBackButton_5:hover {
    background-color: #d7d9c6;
}

/* ======== TABLE VIEW ======== */
QTableView {
    background-color: #ffffff;
    border: 2px solid #b5c58b;
    border-radius: 10px;
    gridline-color: #b5c58b;
    color: #42522b;
    font-size: 14px;
}

QHeaderView::section {
    background-color: #d6d7be;
    color: #3c3c3c;
    padding: 6px;
    border: none;
    font-weight: bold;
}

/* Scrollbars */
QScrollBar:vertical {
    width: 12px;
    background: #e8e8e8;
}
QScrollBar::handle:vertical {
    background: #b5c58b;
    border-radius: 6px;
}
QScrollBar:horizontal {
    height: 12px;
    background: #e8e8e8;
}
QScrollBar::handle:horizontal {
    background: #b5c58b;
    border-radius: 6px;
}
</string>
  </property>
  <widget class="QWidget" name="widget_11" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>571</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,7">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_10">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>761</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QPushButton" name="goBackButton_5">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#goBackButton {
    background-color: white;     /* білий фон */
    color: #333333;              /* темно-сірий текст */
    border: 1px solid #cccccc;   /* легка сіра рамка */
    border-radius: 8px;          /* заокруглені кути */
    padding: 6px 12px;           /* відступи */
}

#goBackButton:hover {
    background-color: #f2f2f2;   /* трохи темніший при наведенні */
}

#goBackButton:pressed {
    background-color: #e6e6e6;   /* ще темніший при натисканні */
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/left-arrow.png</normaloff>icons/left-arrow.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="titleLabel">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>-1</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Return books</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>381</width>
            <height>481</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <widget class="QWidget" name="horizontalLayoutWidget_4">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>371</width>
                <height>51</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="font">
                  <font>
                   <family>Arial Rounded MT</family>
                   <pointsize>-1</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Student name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="studentNameLineEdit">
                 <property name="font">
                  <font>
                   <family>Arial Rounded MT</family>
                   <pointsize>-1</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_6" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <widget class="QWidget" name="horizontalLayoutWidget_5">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>371</width>
                <height>51</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,2">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="font">
                  <font>
                   <family>Arial Rounded MT</family>
                   <pointsize>-1</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Student email</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="studentEmailLineEdit">
                 <property name="font">
                  <font>
                   <family>Arial Rounded MT</family>
                   <pointsize>-1</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <widget class="QWidget" name="horizontalLayoutWidget_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>371</width>
                <height>51</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
               <item>
                <widget class="QLabel" name="label">
                 <property name="font">
                  <font>
                   <family>Arial Rounded MT</family>
                   <pointsize>-1</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Issue ID</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="issueIDLineEdit">
                 <property name="font">
                  <font>
                   <family>Arial Rounded MT</family>
                   <pointsize>-1</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <widget class="QWidget" name="horizontalLayoutWidget_3">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>371</width>
                <height>51</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="font">
                  <font>
                   <family>Arial Rounded MT</family>
                   <pointsize>-1</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Book ID</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="bookIDLineEdit">
                 <property name="font">
                  <font>
                   <family>Arial Rounded MT</family>
                   <pointsize>-1</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_7" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <widget class="QWidget" name="horizontalLayoutWidget_6">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>371</width>
                <height>51</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="font">
                  <font>
                   <family>Arial Rounded MT</family>
                   <pointsize>-1</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Book title</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="bookTitleLineEdit">
                 <property name="font">
                  <font>
                   <family>Arial Rounded MT</family>
                   <pointsize>-1</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_8" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <widget class="QWidget" name="horizontalLayoutWidget_7">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>371</width>
                <height>51</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,2">
               <item>
                <widget class="QLabel" name="label_13">
                 <property name="font">
                  <font>
                   <family>Arial Rounded MT</family>
                   <pointsize>-1</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Author</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="authorLineEdit">
                 <property name="font">
                  <font>
                   <family>Arial Rounded MT</family>
                   <pointsize>-1</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_9" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <widget class="QWidget" name="horizontalLayoutWidget_8">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>371</width>
                <height>51</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,2">
               <item>
                <widget class="QLabel" name="label_14">
                 <property name="font">
                  <font>
                   <family>Arial Rounded MT</family>
                   <pointsize>-1</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Issue date</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="issueDateLineEdit">
                 <property name="font">
                  <font>
                   <family>Arial Rounded MT</family>
                   <pointsize>-1</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_10" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <widget class="QWidget" name="horizontalLayoutWidget_9">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>371</width>
                <height>51</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QPushButton" name="returnButton">
                 <property name="font">
                  <font>
                   <family>Arial Rounded MT</family>
                   <pointsize>-1</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Return</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="clearButton">
                 <property name="font">
                  <font>
                   <family>Arial Rounded MT</family>
                   <pointsize>-1</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Clear</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widget_12" native="true">
          <property name="geometry">
           <rect>
            <x>380</x>
            <y>0</y>
            <width>391</width>
            <height>491</height>
           </rect>
          </property>
          <widget class="QTableView" name="tableView">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>391</width>
             <height>481</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Arial Rounded MT</family>
             <pointsize>-1</pointsize>
             <bold>true</bold>
            </font>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="goBackButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>130</y>
     <width>60</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/left-arrow.png</normaloff>icons/left-arrow.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
